### PR TITLE
Add event log

### PR DESCRIPTION
**Pull request by danielroseman**
Add database, with a bare-bones EventLog model which will form the basis for the transaction log.

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -2,9 +2,9 @@
 
 require "rails"
 # Pick the frameworks you want:
-# require "active_model/railtie"
+require "active_model/railtie"
```
Comment (elliotcm) [inline config/application.rb]: Fairly sure you don't need this when you have active record, and in any case this app has no forms.

File: spec/lib/event_logger_spec.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  }
+
+  it "records an event, given the name, payload and user id" do
+    EventLogger.new.log(action, user_uid, payload)
+    expect(Event.count).to eq(1)
+    expect(Event.first.action).to eq(action)
+    expect(Event.first.user_uid).to eq(user_uid)
+    expect(Event.first.payload).to eq(payload)
+  end
+
```
Comment (elliotcm) [inline spec/lib/event_logger_spec.rb]: Whitespace.

**Comment by elliotcm** [comment]
Couple of minor things but otherwise :+1:

File: config/application.rb (hunk 1 above)
Comment (danielroseman) [inline config/application.rb]: Removed.

File: spec/lib/event_logger_spec.rb (hunk 2 above)
Comment (danielroseman) [inline spec/lib/event_logger_spec.rb]: Fixed.

**Pull request merged**